app/src/test/java/ch/epfl/qedit/backend/MockDBTest.java

```diff
@@ -0,0 +1,133 @@
+package ch.epfl.qedit.backend;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+
+import ch.epfl.qedit.backend.database.DatabaseService;
+import ch.epfl.qedit.backend.database.MockDBService;
+import ch.epfl.qedit.model.Question;
+import ch.epfl.qedit.model.Quiz;
+import ch.epfl.qedit.util.Callback;
+import ch.epfl.qedit.util.Response;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import org.junit.Test;
+
+public class MockDBTest {
+    MockDBService db = new MockDBService();
+    private CountDownLatch lock = new CountDownLatch(1);
+    private String title;
+    private Quiz quiz;
+    private List<Question> questions;
+    private int error;
+
+    private Question bananaQuestion_en =
+            new Question("Banana", "How many banana can you count ?", "matrix1x1");
+
+    private void lockWait(long time) {
+        try {
+            lock.await(time, TimeUnit.MILLISECONDS);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    public void testGetTitle() {
+
+        db.getQuizTitle(
+                "quiz0",
+                new Callback<Response<String>>() {
+                    @Override
+                    public void onReceive(Response<String> data) {
+                        title = data.getData();
+                        error = data.getError();
+                    }
+                });
+        lockWait(2100);
+        assertEquals(Response.NO_ERROR, error);
+        assertEquals("I am a Mock Quiz!", title);
+    }
+
+    @Test
+    public void testGetQuestions() {
+
```

Comment:
Extra space here but no extra space at the end of the function but you don't even do it in other part of the code ? Lack of consistent behavior in this PR

Reply:
I agree with you, but you don't need to be aggressive ;). I am sure he will be more open to your feedback if you don't blame him on a space in his code -_-.